pom.xml

```
@@ -20,7 +20,7 @@
         <connection>scm:git:git://github.com/jenkinsci/validating-string-parameter-plugin.git</connection>
         <developerConnection>scm:git:git@github.com:jenkinsci/validating-string-parameter-plugin.git
         </developerConnection>
-        <url>http://github.com/jenkinsci/validating-string-parameter-pluginn</url>
+        <url>http://github.com/jenkinsci/validating-string-parameter-plugin</url>
```

Please use an HTTPS URL rather than the HTTP URL.

```suggestion
        <url>https://github.com/jenkinsci/validating-string-parameter-plugin</url>
```

---

I made a new PR with HTTPS here https://github.com/jenkinsci/validating-string-parameter-plugin/pull/17

---

I think that this change and #17 will make the documentation displayed on https://plugins.jenkins.io/validating-string-parameter/ much worse.  I believe it will switch plugins.jenkins.io to show the contents of the README.txt file in this repository.

Could you use the [wiki migration instructions](https://www.jenkins.io/doc/developer/publishing/wiki-page/#migrating-from-wiki-to-github) to migrate the plugin documentation from the wiki to a README.adoc or README.md in the plugin repository?  It is also described in a [blog post](https://www.jenkins.io/blog/2019/10/21/plugin-docs-on-github/).